compiler/src/iree/compiler/Dialect/Flow/Transforms/test/outline_dispatch_regions.mlir

```diff
@@ -205,7 +205,7 @@ func.func @main() -> tensor<4x8xf32> {
 
 //      CHECK: flow.executable private @main_dispatch_0 {
 // CHECK-NEXT:   flow.executable.export public @main_dispatch_0_fill_40
-//      CHECK: func.func @main_dispatch_0_fill_40(
+//      CHECK: func.func @main_dispatch_0_fill_40_f32(
```

Comment:
Please also add a new test for `i32xi32xf32` (or some other combination of types). There are a few tests in this file, each of which tries to be fairly minimal (well, as minimal as linalg ops can be), so I'd advise against just copying a dispatch from a real program.